### ISSUE TITLE
Picker causes infinite scroll on touch screen in .NET MAUI

### ISSUE THREAD

**Issue by VictoriaRamirezCharles** [dotnet/maui#24718]
### Description

When interacting with a `Picker` on a touch screen device, the list of items becomes infinitely scrollable. 

### Steps to Reproduce

1. Create a new .NET MAUI project.
2. Add a `Picker` with a list of items using bindings.
3. Run the application on an device with a touch screen.
4. Tap on the `Picker` to select an item.

**Expected Behavior**:
The `Picker` should open and display a list of items that can be scrolled and selected normally.

**Actual Behavior**:
The `Picker` becomes infinitely scrollable.

**Additional Info**:
The issue seems to only happen on devices with touch screens.

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows

**Comment by mattleibow** [comment]
What do you mean by infinitely scrollable? 

* the list just goes on forever even though you only have 5 items? What are the other items?
* the list loops around so there is no start/end but just a loop of the same items?

Maybe add a screen recording of what you mean if I got it wrong so we can have a look.

**Comment by VictoriaRamirezCharles** [comment]
> What do you mean by infinitely scrollable?
> 
> * the list just goes on forever even though you only have 5 items? What are the other items?
> * the list loops around so there is no start/end but just a loop of the same items? (this may be a platform/OS design choice)
> 
> Maybe add a screen recording of what you mean if I got it wrong so we can have a look.

Hello, I made an update to my report and uploaded a video. I realized what happens when I have more than seven elements.

**Comment by mattleibow** [comment]
This is the source from the WinUI 3 code: https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/release/1.6.0/src/controls/dev/ComboBox/ComboBox_themeresources.xaml#L589-L595

>             <Popup x:Name="Popup">
>               <Border x:Name="PopupBorder" Background="{ThemeResource ComboBoxDropDownBackground}" BackgroundSizing="InnerBorderEdge" BorderBrush="{ThemeResource ComboBoxDropDownBorderBrush}" BorderThickness="{ThemeResource ComboBoxDropdownBorderThickness}" Margin="0,-0.5,0,-1" Padding="{ThemeResource ComboBoxDropdownBorderPadding}" HorizontalAlignment="Stretch" CornerRadius="{ThemeResource OverlayCornerRadius}">
>                 <ScrollViewer x:Name="ScrollViewer" Foreground="{ThemeResource ComboBoxDropDownForeground}" MinWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.DropDownContentMinWidth}" VerticalSnapPointsType="OptionalSingle" VerticalSnapPointsAlignment="Near" HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}" HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}" VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}" VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}" IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}" IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}" IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}" BringIntoViewOnFocusChange="{TemplateBinding ScrollViewer.BringIntoViewOnFocusChange}" ZoomMode="Disabled" AutomationProperties.AccessibilityView="Raw">
>                   <ItemsPresenter Margin="{ThemeResource ComboBoxDropdownContentMargin}" />
>                 </ScrollViewer>
>               </Border>
>             </Popup>

Not sure what makes it loop. Does this happen in a plain WinUI 3 app? I think this is an OS choice.

**Comment by VictoriaRamirezCharles** [comment]
> This is the source from the WinUI 3 code: https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/release/1.6.0/src/controls/dev/ComboBox/ComboBox_themeresources.xaml#L589-L595
> 
> > ```
> >         <Popup x:Name="Popup">
> >           <Border x:Name="PopupBorder" Background="{ThemeResource ComboBoxDropDownBackground}" BackgroundSizing="InnerBorderEdge" BorderBrush="{ThemeResource ComboBoxDropDownBorderBrush}" BorderThickness="{ThemeResource ComboBoxDropdownBorderThickness}" Margin="0,-0.5,0,-1" Padding="{ThemeResource ComboBoxDropdownBorderPadding}" HorizontalAlignment="Stretch" CornerRadius="{ThemeResource OverlayCornerRadius}">
> >             <ScrollViewer x:Name="ScrollViewer" Foreground="{ThemeResource ComboBoxDropDownForeground}" MinWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.DropDownContentMinWidth}" VerticalSnapPointsType="OptionalSingle" VerticalSnapPointsAlignment="Near" HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}" HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}" VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}" VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}" IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}" IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}" IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}" BringIntoViewOnFocusChange="{TemplateBinding ScrollViewer.BringIntoViewOnFocusChange}" ZoomMode="Disabled" AutomationProperties.AccessibilityView="Raw">
> >               <ItemsPresenter Margin="{ThemeResource ComboBoxDropdownContentMargin}" />
> >             </ScrollViewer>
> >           </Border>
> >         </Popup>
> > ```
> 
> Not sure what makes it loop. Does this happen in a plain WinUI 3 app? I think this is an OS choice.

Yes, it also happens in WinUI 3. I uploaded 2 repositories with both scenarios and I updated my question to include them, I also added videos.

**Comment by mattleibow** [comment]
This more feels like an OS decision, so maybe they have a way to disable it. Are you able to open an issue in that repo or ask on SO/discord?

**Comment by lothrop** [comment]
We're being bitten by this bug. It was filed as on issue at https://github.com/microsoft/microsoft-ui-xaml/issues/9910 over a year ago, but nobody's responded.

**Comment by pme442** [comment]
While testing on an actual windows tablet I just discovered this problem, too.  Come on.
